src/backend/distributed/utils/metadata_cache.c

```diff
@@ -339,10 +365,12 @@ CitusHasBeenLoaded(void)
 Oid
 DistShardRelationId(void)
 {
-	static Oid cachedOid = InvalidOid;
+	Oid cachedOid = metadataCacheState.distShardRelationId;
```

Comment:
Okay, I'll add a commit which removes the struct. Should these functions still create a local variable , or are we okay with having the functions just read and write from their respective globals?